pubspec.yaml

```diff
@@ -12,6 +12,7 @@ dependencies:
     # path: system_theme_web/
   flutter:
     sdk: flutter
+  equatable:
```

Comment:
This is no longer needed on the lastest master

Reply:
In my implementation the SystemAccentColor class now extends Equatable. This is necessary if you want debounce events from stream using Distinct(), so eliminating unnecessary rebuilds when Win32 send same message several times. It will happen when you switch theme in windows, still don't know why. The alternative can be to check for real differences on the platform side, prior to send the  event via platform channels.